CHANGELOG.md

```diff
@@ -1,3 +1,1034 @@
+<a name="1.6.0-rc.0"></a>
+# 1.6.0-rc.0 bracing-vortex (Tue Oct 25 2016)
+
+## Major notes
+Please read the [Sandbox Removal Blog Post](http://angularjs.blogspot.com/2016/09/angular-16-expression-sandbox-removal.html).
+
+## Bug Fixes
+- **input:** fix `step` validation for `input[number]`/`input[type=range]` ([081d06](https://github.com/angular/angular.js/commit/081d06ffd15c2c6c539ce97b5eb63fa8e2403818) [#15257](https://github.com/angular/angular.js/issues/15257))
```

Comment:
Since you are changing `input[xyz]` to `input[type=xyz]`: You missed one here (`input[number]`) :smiley: